app/models/open_conference_ware/user.rb

```diff
@@ -256,6 +256,14 @@ def sessions
       return self.proposals.confirmed
     end
 
+    # For use in merging duplicate user records
+    def take_associations_from(dup)
+      dup.authentications.each { |a| self.authentications << a }
```

Comment:
To avoid looping, I think these could be written as: `self.authentications += dup.authentications`